css/sakura-dark.css

```diff
@@ -221,3 +221,7 @@ label, legend, fieldset {
   margin-bottom: 0.5rem;
   font-weight: 600;
 }
+
+div {
+   background-color: #4a4a4a
+}
```

Comment:
The CSS is built from [these source files](https://github.com/oxalorg/sakura/tree/master/scss), so you should probably modify those instead.